.eslintrc.cjs

```diff
@@ -1,11 +1,16 @@
 module.exports = {
   root: true,
   parser: "@typescript-eslint/parser",
-  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
```

Comment:
I personally like to keep linting and formatting separate - I'm not going to be super hands-on moving forward and no longer "own" this repo so this isn't blocking feedback but I wanted to mention it.

Reply:
To be honest, I didn't know what this setting did when I added it, and I just copied it from the SvelteKit template. After some Googling, it seems as though placing `prettier` in this file enforces a linting and formatting separation, based on [this docs page](https://prettier.io/docs/en/integrating-with-linters.html) and because this PR adds `eslint-config-prettier` to the package.json.